### DISCUSSION TITLE
At a loss with MacOS

### DISCUSSION THREAD

**Discussion by SouthWestie** [DistroAV/DistroAV#792] [Q&A - Support]
I am currently trying to pass an NDI stream from my PC to my MacBook to stream on twitch with the encoding done on the Mac.

Every time I run OBS 28.1.2 I get an error "NDI Runtime not found" I've done everything I can to fix the problem yet it still persists.
I'm using all the latest releases of the OBS NDI plugin. 

Any help would be much appreciated!

**Comment by Trouffman** [comment]
Read installation instruction again :)
